frontend/tests/components/admin/droplet-client.test.tsx

```
@@ -1,13 +1,33 @@
-import { render, screen, fireEvent, act } from "@testing-library/react";
+import {
+  render,
+  screen,
+  fireEvent,
+  act,
+  waitFor,
+} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 jest.mock("@/components/admin/droplets/droplet-block", () => ({
   DropletBlock: ({ droplet }: { droplet: any }) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)